backend-rust/src/indexer.rs

```diff
@@ -932,121 +932,13 @@ WHERE blocks.finalization_time IS NULL AND blocks.height <= last.height
 }
 
 struct PreparedAccountStatement {
-    account_address:  String,
+    account_address:  Arc<String>,
```

Comment:
I do not really get the benefit of this but sure.

Reply:
The point is to avoid allocating strings everywhere when cloning, this will instead point to the same allocation and should reduce memory usage.
But I'll admit that I don't have any numbers to back this up.

Reply:
atleast it is consistent.